packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -336,7 +335,6 @@ const AuthAndAccountSetupRoutes = ({
 
       {/* Other */}
       <Clear path="/clear/*" />
-      <CookiesDisabled path="/cookies_disabled/*" />
```

Comment:
We may want to keep this and redirect (to email-first?) if local storage is enabled?

Reply:
> We may want to keep this and redirect (to email-first?) if local storage is enabled?

I _believe_ not redirecting is consistent with the behaviour in Backbone.